case_studies/gauss_siedel/galewsky_serial.py

```diff
@@ -0,0 +1,279 @@
+from firedrake.petsc import PETSc
```

Comment:
It's Seidel not Siedel (German vowel combinations)

Reply:
https://github.com/firedrakeproject/asQ/pull/164/commits/0242daac22353abb476dec80931d80281a3b8aa9